### ISSUE TITLE
plugin.toml no such file or directory

### ISSUE THREAD

**Issue by austinpray** [statianzo/dokku-supervisord#30]
Getting this message a lot whenever I run any dokku commands. `dokku ps:rebuildall` and `dokku version` for instance.

```
2016/02/11 20:07:20 open /var/lib/dokku/plugins/available/supervisord/plugin.toml: no such file or directory
```

Here is how the server was configured exactly: http://dokku.viewdocs.io/dokku/getting-started/install/linode/

Installed plugin with: `dokku plugin:install https://github.com/statianzo/dokku-supervisord.git`

Dokku version: `0.4.13`

I guess this plugin isn't compatible with 0.4.x?

**Comment by kenips** [comment]
Correct. 0.4.x requires `plugin.toml` which we don't have. dokku supports running multiple processes defined in Procfile since 0.3.14 and as a result I've moved over to use Procfile too. Why do you need `supervisord`?

**Comment by austinpray** [comment]
1) out of habit, I used this plugin on 0.3.x.
2) automatic crash recovery. Although it looks like this too might have been added to dokku? https://github.com/dokku/dokku/commit/ab8eb5a727d2c77c017f49c95bbd873203cab06e

**Comment by kenips** [comment]
Yeah, looks like that's the migration path for us. Of course whether or not to support `supervisord` beyond 0.3.x is up to @statianzo.  I was just curious if there's a strong use case for it.

**Comment by statianzo** [comment]
Given that dokku has a comparable feature to this plugin, I don't have plans of trying to make it compatible. I'll update the readme to reflect that.

**Comment by statianzo** [comment]
Given that dokku has a comparable feature to this plugin, I don't have plans of trying to make it compatible. I'll update the readme to reflect that.

**Comment by statianzo** [comment]
Updated the readme with details about dokku 0.4 incompatibility, linking to [dokku-logging-supervisord](https://github.com/sehrope/dokku-logging-supervisord) for those interested in sticking with supervisord. Thanks @kenips and @austinpray for bringing the incompatibility to my attention.
